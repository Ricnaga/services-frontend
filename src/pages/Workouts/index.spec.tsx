import { render } from '@testing-library/react';
import { Workouts } from '.';

describe('WORKOUTS SCREEN', () => {
  it('should be able to render screen', () => {
    const { container } = render(<Workouts />);

    expect(container).toMatchSnapshot();
  });
});
