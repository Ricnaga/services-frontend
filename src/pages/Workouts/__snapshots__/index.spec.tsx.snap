// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WORKOUTS SCREEN should be able to render screen 1`] = `
<div>
  <div
    class="mt-4 px-4 container"
  >
    <ul
      class="mb-3 nav nav-tabs"
      role="tablist"
    >
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="react-aria6216987408-1-tabpane-estatistica"
          aria-selected="true"
          class="nav-link active"
          data-rr-ui-event-key="estatistica"
          id="react-aria6216987408-1-tab-estatistica"
          role="tab"
          type="button"
        >
          Estatística
        </button>
      </li>
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="react-aria6216987408-1-tabpane-alterar"
          aria-selected="false"
          class="nav-link"
          data-rr-ui-event-key="alterar"
          id="react-aria6216987408-1-tab-alterar"
          role="tab"
          tabindex="-1"
          type="button"
        >
          Alterar serviços
        </button>
      </li>
    </ul>
    <div
      class="tab-content"
    >
      <div
        aria-labelledby="react-aria6216987408-1-tab-estatistica"
        class="fade tab-pane active show"
        id="react-aria6216987408-1-tabpane-estatistica"
        role="tabpanel"
      >
        <div
          class="w-100 mx-auto card"
        >
          <div
            class="card-body"
          >
            <div
              class="placeholder-glow card-title h5"
            >
              <span
                class="w-100 mx-auto my-2 placeholder"
              />
            </div>
            <p
              class="placeholder-glow card-text"
            >
              <span
                class="w-100 my-2 placeholder"
              />
              <span
                class="w-100 my-2 placeholder"
              />
              <span
                class="w-100 my-2 placeholder"
              />
              <span
                class="w-100 my-2 placeholder"
              />
            </p>
            <div
              class="placeholder-glow card-footer"
            >
              <span
                class="w-100 my-2 placeholder"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        aria-labelledby="react-aria6216987408-1-tab-alterar"
        class="fade tab-pane"
        id="react-aria6216987408-1-tabpane-alterar"
        role="tabpanel"
      >
        <div
          class="mt-4 d-flex justify-content-center container"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="card"
              >
                <h1
                  class="card-header"
                >
                  Alterar serviço do cliente
                </h1>
                <div
                  class="px-4 card-body"
                >
                  <form
                    action="#"
                    class="text-center mt-3"
                  >
                    <div
                      class="mb-2"
                    >
                      <input
                        class="form-control"
                        name="id"
                        placeholder="id"
                        type="text"
                        value=""
                      />
                      <small
                        class="text-muted form-text"
                      >
                        Preencha o 
                        id
                      </small>
                    </div>
                    <div
                      class="mb-2"
                    >
                      <input
                        class="form-control"
                        name="nome"
                        placeholder="nome"
                        type="text"
                        value=""
                      />
                      <small
                        class="text-muted form-text"
                      >
                        Preencha o 
                        nome
                      </small>
                    </div>
                    <div
                      class="mb-2"
                    >
                      <input
                        class="form-control"
                        name="email"
                        placeholder="email"
                        type="text"
                        value=""
                      />
                      <small
                        class="text-muted form-text"
                      >
                        Preencha o 
                        email
                      </small>
                    </div>
                    <div
                      class="mb-2"
                    >
                      <input
                        class="form-control"
                        name="rg"
                        placeholder="rg"
                        type="text"
                        value=""
                      />
                      <small
                        class="text-muted form-text"
                      >
                        Preencha o 
                        rg
                      </small>
                    </div>
                    <div
                      class="mb-2"
                    >
                      <input
                        class="form-control"
                        name="endereco"
                        placeholder="endereco"
                        type="text"
                        value=""
                      />
                      <small
                        class="text-muted form-text"
                      >
                        Preencha o 
                        endereco
                      </small>
                    </div>
                    Conta ativa
                    <input
                      class="form-check-input mx-2"
                      name="conta"
                      type="checkbox"
                      value="false"
                    />
                    <div
                      class="mt-3 d-flex justify-content-end card-footer"
                    >
                      <button
                        class="btn btn-outline-warning btn-lg"
                        type="submit"
                      >
                        Buscar
                      </button>
                    </div>
                  </form>
                </div>
              </div>
            </div>
          </div>
          <div
            class="toast-container position-absolute bottom-0 start-0 p-3"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
